### ISSUE TITLE
choco search sorts by title but shows package names

### ISSUE THREAD

**Issue by aronovgj** [chocolatey/choco#256]
either sort by package name or show the titles instead of the package names. or show both.
showing titles would be kind of useless...

Conversation:

![image](https://user-images.githubusercontent.com/12760779/141298090-baf59fba-5ddf-45f5-8da2-3d406eb8f2cc.png)

UPDATE: As Chocolatey Gitter chat is now unavailable, I've added a screenshot of the Gitter chat.

**Comment by ferventcoder** [comment]
Added link to convo.

**Comment by bartvanandel** [comment]
I've created an implementation for this, which adds several options for ordering the results, and sorts by name by default. Unfortunately due to [a bug in NuGet.Server-chocolatey](https://github.com/chocolatey/NuGet.Server-chocolatey/issues/2), the results may still appear slightly out-of-order.

See my branch here: https://github.com/bartvanandel/choco/tree/feat/order-by-parameter

**Comment by bartvanandel** [comment]
It's nice to see that my PR for this issue has been pretty much ignored for over 2 years [/sarcasm]

Truthfully though, this is one of the reasons that people hesitate to open source projects. Maintainers and sometimes complain that if you want something, just contribute yourself, but if a contribution with a genuine fix gets ignored for so long, this game no longer plays out.

**Comment by bartvanandel** [comment]
I give up. For more than 3 I've tried to get this issue and it's accompanying PR reviewed and merged, but @ferventcoder doesn't seem interested enough.

Fortunately I've found a better alternative to Chocolatey in the form of [Scoop](https://scoop.sh). I'd advice everyone to try it, IMO it's better in pretty much every way: it has a larger package repo (especially if you enable the extra buckets that exist), packages use the original (portable) installers, don't require admin privileges by default (default install location is user dir, global override is available), are more easy to update and maintain, and therefore are actually way better maintained.

Goodbye Chocolatey. It was nice knowing you, but you've been outrun.
